.travis.yml

```diff
@@ -1,6 +1,9 @@
 language: php 
 
-sudo: false
+env:
+  global:
+    - COMPOSER_ROOT_VERSION=2.0.x-dev
+    - DB=MYSQL CORE_RELEASE=3.5
```

Comment:
We don't need this line, and the previous line's constraint should match that in composer.json (`2.x-dev`)